libqtile/backend/base.py

```diff
@@ -253,6 +250,54 @@ def info(self) -> dict[str, Any]:
         """
         return {}
 
+    @expose_command()
+    def keep_above(self, enable: bool | None = None):
+        """Keep this window above all others"""
+
+    @expose_command()
+    def keep_below(self, enable: bool | None = None):
+        """Keep this window below all others"""
+
+    @expose_command()
+    def move_up(self, force: bool = False) -> None:
+        """
+        Move this window above the next window along the z axis.
+
+        Will not raise a window where "keep_below" is True unless
+        force is set to True.
```

Comment:
If window is normal and only other window is `kept_above`, this will *not* move above it, even if `force=True` so it's the first one. I'll make this clearer in the code.